src/main/java/com/example/provider/SampleServletModule.java

```diff
@@ -35,6 +35,7 @@ protected void configureServlets() {
     install(new GrpcServiceModule());
 
     serve("/vehicle/*").with(VehicleServlet.class);
+    serve("/vehicles/*").with(VehiclesServlet.class);
```

Comment:
should we rollback these change as well as the 'VehiclesServlet.java' file?

Reply:
hey Arturo, VehiclesServlet.java is the servlet that specifically handles the case when we request for a list of vehicles with specific vehicle attributes.